Components/SwagImportExport/DbAdapters/OrdersDbAdapter.php

```diff
@@ -443,6 +443,7 @@ public function getDefaultColumns()
         ];
 
         $billingColumns = [
+            'billing.number as customerNumber',
```

Comment:
wouldn't it make more sense to put this into the `$customerColumns` array?

Reply:
You are right. Just updated the commit.